app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -89,7 +91,7 @@ public void updateMapEntity(GeoEntityNotification geoEntityNotification) {
 
     @Override
     public void setOnEntityClickedListener(MapEntityClickedListener mapEntityClickedListener) {
```

Comment:
Not part of your new code, but what happens when we need more than one listener?
I think we should change it to AddOnEntityClickedListener.